x-pack/plugins/enterprise_search/server/lib/pipelines/create_pipeline_definitions.ts

```diff
@@ -142,6 +142,7 @@ export const createIndexPipelineDefinitions = async (
                 },
               },
             ],
+            // Commenting for code review
             pattern: '�',
```

Comment:
The scanner doesn't like this pattern because it's not utf-8.  Is this replacement expected?

Reply:
Hah, yes. This pipeline extracts binary content (eg: .pdf, .doc, .rtf) and places its contents in the body tag. But that extraction often results in many useless replacement characters (`�`), which this processor removes. So this is not a case of a character having been replaced with the replacement character �,  but a rare case of using the � in a functional manner.

Reply:
Thanks for the info!